### PR TITLE
add model files

### PR DESCRIPTION
**Pull request by stringtheory1234**

### CODE REVIEW

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/ShopOwner.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ShopOwner {
+    private String shopOwnerUid;
+    private String shopOwnerName;
+}
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/ShopOwner.java:29]: Nit. newline at the end.

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+public class Token {
+
+    enum Status {
+        STATUS_UNKNOWN, ACTIVE, EXPIRED, CANCELLED_BY_CUSTOMER, CANCELLED_BY_SHOP_OWNER;
+    }
+
+    private Integer shopId;
+    private Integer tokenNumber;
+    private Status status;
+    private UUID tokenId;
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java:37]: Nit. Should we make this the first field?

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Token {
+
+    enum Status {
+        STATUS_UNKNOWN, ACTIVE, EXPIRED, CANCELLED_BY_CUSTOMER, CANCELLED_BY_SHOP_OWNER;
+    }
+
+    private Integer shopId;
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java:34]: Why is this an integer?

File: backendservice/pom.xml [hunk 4]
```diff
@@ -52,4 +57,4 @@
 		</plugins>
 	</build>
 
-</project>
+</project>
```
Comment (priytanvani) [inline backendservice/pom.xml:60]: Nit. Add newline at the end?

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Shop.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+@Data
+@NoArgsConstructor
+public class Shop {
+    private UUID shopId;
+    private String shopOwnerUid;
+    private String shopName;
+    private String address;
+    private String phoneNumber;
+    private Integer lastTokenAllotted;
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Shop.java:32]: Do we want this here?

File: backendservice/pom.xml (hunk 4 above)
Comment (stringtheory1234) [reply backendservice/pom.xml]: Done!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Shop.java (hunk 5 above)
Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Shop.java]: Done!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Token {
+
+    enum Status {
+        STATUS_UNKNOWN, ACTIVE, EXPIRED, CANCELLED_BY_CUSTOMER, CANCELLED_BY_SHOP_OWNER;
+    }
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java:32]: Nit. Maybe move this just before the status field?

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java [hunk 7]
```diff
@@ -26,13 +26,11 @@
...
 public class Token {
-
-    enum Status {
-        STATUS_UNKNOWN, ACTIVE, EXPIRED, CANCELLED_BY_CUSTOMER, CANCELLED_BY_SHOP_OWNER;
-    }
-
     private UUID tokenId;
     private UUID shopId;
     private Integer tokenNumber;
+    enum Status {
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java:32]: Nit. Add a newline before this?

**Pull request merged**